dxr/plugins/clang/menus.py

```diff
@@ -201,6 +203,33 @@ def _more_menu_items(self, (qualname,)):
 class FunctionRef(_RefWithDefinition):
     """Ref for function definitions or references"""
 
+    def menu_items(self):
+        """Return a jump-to-definition menu item along with whatever others
+        _more_menu_items() returns.
+
+        """
+        path = self.menu_data[0]
```

Comment:
It doesn't look like `menu_data[0]` is really used here (beyond acting as a boolean), nor is `menu_data[1]`. So that's needlessly inflating the ES indices. There might not be good reason to subclass `_RefWithDefinition` anymore.